From eca9b91b832dfc27d0e1a6d08e2ce4878f229373 Mon Sep 17 00:00:00 2001
From: Jordan Yelloz <jordan@yelloz.me>
Date: Fri, 24 Jun 2016 12:18:45 -0700
Subject: [PATCH] extensions: added AR substitution

This is to ensure that the correct AR is run in cross-compile jobs.
Often a cross-compile build will succeed without this change but it
fails on my Gentoo Linux system when I have binutils installed with the
"multitarget" USE flag.
This change substitues AR with the autotools-supplied AR for the
extensions subdirectory.

Signed-off-by: Jordan Yelloz <jordan@yelloz.me>
Signed-off-by: Pablo Neira Ayuso <pablo@netfilter.org>
---
 extensions/GNUmakefile.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/extensions/GNUmakefile.in b/extensions/GNUmakefile.in
index 53be2cd..33b06ef 100644
--- a/extensions/GNUmakefile.in
+++ b/extensions/GNUmakefile.in
@@ -11,6 +11,7 @@ libdir       = @libdir@
 libexecdir   = @libexecdir@
 xtlibdir     = @xtlibdir@
 
+AR                 = @AR@
 CC                 = @CC@
 CCLD               = ${CC}
 CFLAGS             = @CFLAGS@
-- 
2.7.4

